### PR TITLE
Ran code through formatter and corrected flake8

### PR DESCRIPTION
**Pull request by retr0h**
We use Molecule and Molecule will lint this plugin.  Rather than
exempting this library, figured we could correct the flake8 errors.

### CODE REVIEW

File: library/pdns_zone.py [hunk 1]
```diff
@@ -144,13 +150,11 @@
           api_port={{ api_port }}
 '''
 
-headers = {
```
Comment (jpmens) [inline library/pdns_zone.py:147]: Why are you changing this formatting? I find the single-line version, while terse, doesn't add clarity.

File: library/pdns_zone.py [hunk 2]
```diff
@@ -191,7 +197,9 @@ def zone_exists(module, base_url, zone):
         return None
 
     if info['status'] != 200:
-        module.fail_json(msg="failed to check zone %s at %s: %s" % (zone, url, info['msg']))
```
Comment (jpmens) [inline library/pdns_zone.py:194]: why not single line? Here I find it is clearer ...
(there are a few of these further)

File: library/pdns_zone.py [hunk 3]
```diff
@@ -209,19 +218,24 @@ def zone_list(module, base_url, zone=None):
...
-        module.fail_json(msg="failed to enumerate zones at %s: %s" % (url, info['msg']))
+        module.fail_json(
+            msg="failed to enumerate zones at %s: %s" % (url, info['msg'])
+        )
 
     content = response.read()
     data = json.loads(content)
     for z in data:
         if zone is None or fnmatch.fnmatch(z['name'], zone):
-            list.append({
```
Comment (jpmens) [inline library/pdns_zone.py:218]: what's wrong with this?

File: library/pdns_zone.py [hunk 4]
```diff
@@ -244,30 +260,44 @@ def zone_add_slave(module, base_url, zone, masters, comment):
...
     if kind == 'MASTER' or kind == 'NATIVE':
-        module.fail_json( msg="zone %s is %s. Cannot convert to slave" % (zone, kind))
+        module.fail_json(
+            msg="zone %s is %s. Cannot convert to slave" % (zone, kind)
+        )
 
     masters = masters.split(',')
 
     data = {
-        'kind'      : 'Slave',
```
Comment (jpmens) [inline library/pdns_zone.py:252]: why the reformatting here?

File: library/pdns_zone.py [hunk 5]
```diff
@@ -244,30 +260,44 @@ def zone_add_slave(module, base_url, zone, masters, comment):
...
     if info['status'] != 201:
-        module.fail_json(msg="failed to create slave zone %s at %s: %s" % (zone, base_url, info['msg']))
+        module.fail_json(
+            msg="failed to create slave zone %s at %s: %s"
+            % (zone, base_url, info['msg'])
+        )
 
     return True
 
-def zone_add_master(module, base_url, zone, soa_rdata, ns_rrset, comment, ttl=60, wantkind='Master'):
```
Comment (jpmens) [inline library/pdns_zone.py:270]: I have to ask again: what's wrong here? I don't much care for the parameters on a new line each

File: library/pdns_zone.py [hunk 6]
```diff
@@ -276,55 +306,67 @@ def zone_add_master(module, base_url, zone, soa_rdata, ns_rrset, comment, ttl=60
...
     if kind == 'NATIVE':
         if kind == 'SLAVE' or kind == 'MASTER':
-            module.fail_json( msg="zone %s is %s. Cannot convert to native" % (zone, kind))
+            module.fail_json(
+                msg="zone %s is %s. Cannot convert to native" % (zone, kind)
+            )
 
     records = []
     data = {
-        'kind'          : str(wantkind).lower().title(),
```
Comment (jpmens) [inline library/pdns_zone.py:286]: please undo this reformatting; it was clear. It now no longer is.

File: library/pdns_zone.py [hunk 7]
```diff
@@ -276,55 +306,67 @@ def zone_add_master(module, base_url, zone, soa_rdata, ns_rrset, comment, ttl=60
...
+        'name': zone,
+        # I'm creating records "manually" below to avoid automatic TTL=3600
+        'nameservers': [],
+        'records': records,
+        'comments': [
+            {'name': zone, 'type': 'SOA', 'account': '', 'content': comment}
+        ],
     }
 
-    records.append({
```
Comment (jpmens) [inline library/pdns_zone.py:299]: why the reformatting?

File: library/pdns_zone.py [hunk 8]
```diff
@@ -276,55 +306,67 @@ def zone_add_master(module, base_url, zone, soa_rdata, ns_rrset, comment, ttl=60
...
+            'type': 'SOA',
+            'name': zone,
+            'ttl': ttl,
+            'disabled': False,
+            'content': soa_rdata,
+        }
+    )
 
     for ns in ns_rrset.split(','):
-        records.append({
```
Comment (jpmens) [inline library/pdns_zone.py:308]: here too?

File: library/pdns_zone.py [hunk 9]
```diff
@@ -335,66 +377,78 @@ def main():
...
-    masters   = module.params['masters']
-    action    = module.params['action']
-    soa       = module.params['soa']
-    nsset     = module.params['nsset']
-    comment   = module.params['comment']
-    ttl       = module.params['ttl']
-
-    base_url = 'http://{0}:{1}/api/v1/servers/localhost/zones'.format(api_host, api_port)
+    api_key = module.params['api_key'] if module.params['api_key'] else api_key
+    api_host = (
```
Comment (jpmens) [inline library/pdns_zone.py:403]: these changes make no sense to me; please keep as they were

**Review by jpmens — CHANGES_REQUESTED** [review]
I've added comments inline

**Comment by retr0h** [comment]
Sure, I don't have to run it through a formatter.  I ran the code through [black](https://github.com/python/black).  I'll adjust it back to your liking :)

File: library/pdns_zone.py (hunk 2 above)
Comment (retr0h) [reply library/pdns_zone.py:194]: The issue is line length.

File: library/pdns_zone.py (hunk 5 above)
Comment (retr0h) [reply library/pdns_zone.py:270]: line length

**Comment by retr0h** [comment]
I'll close this out.  I'll use my fork instead.

**Pull request closed without merging**